common/workerpool.c

```diff
@@ -198,7 +198,7 @@ void workerpool_run(workerpool_t *wp)
 
 int workerpool_get_nprocs()
 {
-#ifdef WIN32
+#ifdef WIN32 // "WIN32" is defined through <windows.h> header
```

Comment:
I thought you wanted to use the `_WIN32` macro here to be consistent with the rest of the code base. Do you want to replace the `WIN32` macro of keep it? I am now in favour of using `_WIN32` consistently.

Reply:
Are we sure `SYSTEM_INFO sysinfo;` and following lines could not be specific to a toolchain or something like that?

And it was the intend of the original author to use this specific macro?

---

Otherwise, sure I can update the MR.

Reply:
> And it was the intend of the original author to use this specific macro?

The use of the macros originates from https://github.com/AprilRobotics/apriltag/pull/27. There is no mention of explicitly using the two different macros.

> Are we sure `SYSTEM_INFO sysinfo;` and following lines could not be specific to a toolchain or something like that?

The function [`GetSystemInfo`](https://learn.microsoft.com/en-us/windows/win32/api/sysinfoapi/nf-sysinfoapi-getsysteminfo) also seems to be part of the standard Win32 API.

The CI is testing the GCC, clang and MSVC compilers on Windows. If none of them complains, then I think we are pretty safe in using that macro.